crates/lib/src/qpu/api.rs

```diff
@@ -690,7 +720,10 @@ pub enum QpuApiError {
         /// The message associated with the failed job.
         message: String,
     },
-
+    /// Error that can occur when the gRPC status code could not be decoded.
+    #[error("The status code could not be decoded: {0}")]
+    StatusCodeDecode(String), // TODO: This error is in prost. Should we really use that as a dep
```

Comment:
Stale comment, or is `prost` imported somewhere I'm not seeing?

Reply:
Stale comment, removed